src/_pytest/deprecated.py

```diff
@@ -20,11 +20,11 @@
 YIELD_TESTS = "yield tests were removed in pytest 4.0 - {name} will be ignored"
 
 
-FIXTURE_FUNCTION_CALL = UnformattedWarning(
-    RemovedInPytest4Warning,
-    'Fixture "{name}" called directly. Fixtures are not meant to be called directly, '
-    "are created automatically when test functions request them as parameters. "
-    "See https://docs.pytest.org/en/latest/fixture.html for more information.",
+FIXTURE_FUNCTION_CALL = (
+    'Fixture "{name}" called directly. Fixtures are not meant to be called directly,\n'
+    "but are created automatically when test functions request them as parameters.\n"
+    "See https://docs.pytest.org/en/latest/fixture.html for more information about fixtures, and\n"
+    "https://docs.pytest.org/en/latest/deprecations.html#calling-fixtures-directly about how to update your code."
 )
```

Comment:
should this get inlined now that it's not really "deprecated"? (and only used here)

Reply:
@asottile it helps to stay consistent to have those constants at one place, even if they are used only in one place

als this one should go away once we shift from fixture returning a function to fixture returning the fixturedefinition instance